action.yml

```diff
@@ -50,6 +50,14 @@ inputs:
      Example: `ublue-os`
     required: false
     default: ${{ github.repository_owner }}
+  use_cache:
+    description: |
+      Make use of docker buildx cache. This is an experimental feature of docker buildx
+      so it isn't guaranteed to work.
+      Input must match the string 'true' for the step to be enabled.
+    required: false
+    default: 'false'
```

Comment:
Ok, after some testing, I think we should not have this option quite yet until I can make a patch to better deal with the use of the `docker-container` driver. There is a HUGE slow down when we run with `docker-container` which is required to try caching build contents.

![image](https://github.com/blue-build/github-action/assets/4626052/2ac9ff98-661a-4c4f-bd70-1dc83a67b5aa)

https://github.com/gmpinder/uhypr/actions/workflows/build.yml

Reply:
```suggestion
```